devito/core/cpu.py

```diff
@@ -4,7 +4,8 @@
 from devito.exceptions import InvalidOperator
 from devito.passes.equations import collect_derivatives
 from devito.passes.clusters import (Lift, blocking, buffering, cire, cse,
-                                    extract_increments, factorize, fuse, optimize_pows)
+                                    extract_increments, factorize, fission, fuse,
```

Comment:
Should for homogenity use also fusion instead of fuse ?

Reply:
there's already inconsistency, unfortunately (blocking, buffering, fission ... vs fuse, extract_, optimize_, ...)